src/scripts/main.js

```diff
@@ -1,3 +1,44 @@
 'use strict';
 
-// write code here
+// select all <li> that are direct child of <ul>
+const list = document.querySelectorAll('ul > li');
+
+function sortList(itemsList) {
+  // Return new sorted array
+  // originaly list = [li, li, li], each <li> is an object.
+  // Each <li> object has dataset.
+  // Each dataset has the salary key which stores string (salary value)
+  return Array.from(itemsList).sort((workerA, workerB) => {
+    /* here we praseFloat/praseInt,
+        and replace all non-numerical cahracters with nothing.
```

Comment:
There are typos in the comments: 'praseFloat/praseInt' should be 'parseFloat/parseInt', and 'cahracters' should be 'characters'. These typos do not affect the functionality but should be corrected for clarity.